esmvaltool/recipes/recipe_quantilebias.yml

```diff
@@ -0,0 +1,42 @@
+# recipe_quantilebias.yml
+---
+
+documentation:
+  description: |
+    Tool for calculation of precipitation quantile bias
+
+  authors:
+    - arno_en
+    - hard_jo
+
+  references:
+    - mehran14jgr
+
+  projects:
+    - c3s-magic
+
+datasets:
+  - {dataset: EC-EARTH,       project: CMIP5, exp: historical, ensemble: r1i1p1, start_year: 1997, end_year: 1997}
+
+preprocessors:
```

Comment:
If you only use the default preprocessor, there is no need to define one, you can just leave it out.

Reply:
done